packages/icons/src/library/replace.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * WordPress dependencies
+ */
+import { Path, Rect, SVG } from '@wordpress/primitives';
+
+const replace = (
+	<SVG viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
+		<Rect x="11" y="3" width="7" height="5" rx="1" />
```

Comment:
Usually I flatten the icon entirely to prevent any overlapping elements, that reduces the element to a single Path.

Reply:
Thanks for the tip!